styles/typography/typography-preset-1.json

```diff
@@ -0,0 +1,37 @@
+{
+	"version": 3,
+	"$schema": "https://schemas.wp.org/trunk/theme.json",
+	"title": "Beiruti & Literata",
+	"slug": "typography-preset-1",
+	"styles": {
+		"typography": {
+			"fontFamily": "var:preset|font-family|literata",
+			"fontSize": "var:preset|font-size|medium",
+			"fontStyle": "normal",
+			"fontWeight": "300",
+			"letterSpacing": "-0.22px",
+			"lineHeight": "1.5"
+		},
+		"blocks": {
+			"core/site-title": {
+				"typography": {
```

Comment:
I think the uppercase should be part of the full style variation, the one that creates the complete page design, but not necessarily be used every time the preset is selected. This is difficult to balance without being able to test it.

Reply:
That sounds good to me. Maybe we can go without it for now and iterate later once we have it shaped up. I felt it was worth it to highlight.

Reply:
Yes, site title in this variation should be uppercase.

Reply:
I would like this to be clarified, because I want to make sure that we are talking about the same thing.
-The style variation is the one that you select under Site Editor > Styles > top of the sidebar. This is a combination of block styles, colors, and typography unique for this variation.
-The typography preset is the one that you select under Site Editor > Styles > Typography. This is the typography only, and can be applied on top of any other style variation.

Reply:
Apologies, I was referring to these as style variations. Please ignore the feedback if it doesn't apply.